sponsors/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
@polybuildr , @hharchani , @nisargjhaveri 

I want the sponsors to go on main page and not a copy of /12's page.

Please discuss.